lib/js/components/DatePickers/DatePicker/DatePicker.vue

```diff
@@ -0,0 +1,291 @@
+<template>
+	<div
+		class="ds-datePicker"
```

Comment:
@pioziol123 we need a slot for switch, look at usage https://www.figma.com/design/03ABNCSDYWYDmOPJOBGM5l/INI-153-Planowanie-pracy-z-lekcjami?node-id=1281-190970&m=dev